src/components/Pages/Revision/RevisionPage.js

```diff
@@ -21,14 +21,14 @@ export default class HistoryCaseRevision extends React.Component {
 
   componentDidMount() {
     const { station, caseid } = this.props.match.params;
-    const markScheme = dummyData.history[station][caseid]["mark-scheme"].map(
-      element => ({ text: element, completed: false })
+    const revisionList = dummyData.history[station][caseid]["mark-scheme"].map(
```

Comment:
Given all the amazing work you've done for this PR, I'm going to be a complete cow and say that I don't like this variable name. Not sure what was wrong with `markScheme` -- if we want to be clear it's a list, we could go with `markSchemeList`?